packages/hasura/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@golevelup/nestjs-hasura",
-  "version": "1.3.2",
+  "version": "2.0.0",
```

Comment:
We should revert all of the changes to package.json. These values are managed automatically when we do a release and shouldn't be incremented by hand.

Reply:
I've removed the bumps :)